django_elasticsearch_dsl/fields.py

```diff
@@ -124,7 +124,7 @@ def get_value_from_instance(self, instance, field_value_to_ignore=None):
 
         if objs is None:
             return {}
-        if isinstance(objs, collections.Iterable):
+        if not isinstance(objs, SimpleLazyObject) and isinstance(objs, collections.Iterable):
```

Comment:
I think it can be better written as following so it is more pythonic and readable

```suggestion
       objs_type = type(objs)

        if objs_type is collections.Iterable and objs_type is not SimpleLazyObject:
```